spread.yaml

```diff
@@ -601,6 +601,7 @@ exclude:
     - cmd/autom4te.cache
     - "*.o"
     - "*.a"
+    - "*.pyc"
```

Comment:
hm likely this is the only change we need

Reply:
When running interactively we probably also want the one to tar. That is technically the real fix.